onnx/version_converter/adapters/upsample_6_7.h

```diff
@@ -0,0 +1,38 @@
+// Adapter for Upsample in default domain from version 6 to 7
+
+#pragma once
+
+#include "onnx/version_converter/adapters/adapter.h"
+
+namespace ONNX_NAMESPACE { namespace version_conversion {
+
+struct Upsample_6_7 final: public Adapter {
+  explicit Upsample_6_7() : Adapter("Upsample", OpSetID(6), OpSetID(7)) {}
+
+  void adapt_upsample_6_7(std::shared_ptr<Graph>, Node* node) const {
+
+    Symbol width_scale_symbol = Symbol("width_scale");
+    Symbol height_scale_symbol = Symbol("height_scale");
+    ONNX_ASSERTM(node->hasAttribute(width_scale_symbol) && node->hasAttribute(height_scale_symbol), "Upsample in opset 1 needs to have width_scale and height_scale attributes");
+    
+    auto width_scale = node->f(width_scale_symbol);
+    auto height_scale = node->f(height_scale_symbol);
+    
+    auto input_shape = node->inputs()[0]->sizes();
+    ONNX_ASSERTM(input_shape.size() >= 2, "Upsample in opset 1 needs at least 2D input tensor");
```

Comment:
Is 2 here correct? I thought Upsample-1 (https://github.com/onnx/onnx/blob/master/docs/Changelog.md#upsample-1) needs 4D input tensor?

Reply:
You are right. 4D is easier. Fixed.